_i18n/de/resources/moneropedia/tail-emission.md

```diff
@@ -1,15 +1,15 @@
 ---
-terms: ["Tail-Emission"]
-summary: "the block reward at the end of the emission curve"
+terms: ["Tail-Emission", "Anschlussverteilung"]
```

Comment:
```suggestion
terms: ["Tail-Emission", "End-Emission"]
```

Just  a suggestion.. I think that translation would make sense.. it's the "Emission" miners will get forever once we reach the end. Possible alternative would be "Abschluss-Emission".

Reply:
I think it would make sense to use "Anschluss" instead of "Abschluss", i.e. "Anschluss-Emission" since it's the emission miners get subsequently. "Abschluss" feels a bit too absolute to me.

Reply:
Both Anschluss/Abschluss/End would be fine. It's just that reading "Verteilung" in German doesn't make me think of block rewards or payments in general at all... "Verteilung" is sharing or splitting something up (while miners get the whole block reward). Anyway I see if this has already been decided upon, it'd be another PR..